style.css

```diff
@@ -327,14 +327,14 @@ ul li ul {
     padding: 0 0 0 1.5em;
 }
 dt {
-    width: 140px;
```

Comment:
The rewrite is coming... :smiling_imp: :laughing:

(Not that I've started on it yet..... but soon!)